draft-ietf-gnap-core-protocol.md

```diff
@@ -3778,28 +3792,56 @@ B7_8Wbw4ttzbMS_doJvuDagW8A1Ip3fXFAHtRAcKw7rdI4_Xln66hJxFekpdfWdiPQddQ6Y
 }
 ~~~
 
+The verifier MUST parse and validate the DPoP proof header as defined in
+{{I-D.ietf-oauth-dpop}}. If the HTTP message request includes a message body,
+the verifier MUST calculate the digest of the body and compare it to the
+`htd` value. The verifier MUST ensure the key presented in the DPoP
+proof header is the same as the expected key of the signer.
+
 ### HTTP Message Signing {#httpsig-binding}
 
 This method is indicated by `httpsig` in
-the `proof` field. The client instance creates an HTTP
-Signature header as described in {{I-D.ietf-httpbis-message-signatures}} section 4. The client instance MUST
-calculate and present the Digest header as defined in {{RFC3230}} and include
-this header in the signature.
+the `proof` field. The sender creates an HTTP
+Message Signature as described in {{I-D.ietf-httpbis-message-signatures}}. 
+
+The covered content of the signature MUST include the following:
+
+@request-target:
+: the target of the HTTP request 
+
+digest:
+: The Digest header as defined in {{RFC3230}}. When the request message has a body, 
+the signer MUST calculate this header value and the verifier 
+MUST validate this header.
+
+When the request is bound to an access token, the covered content
+MUST also include:
+
+authorization:
+: The Authorization header used to present the access token as discussed in
+{{use-access-token}}.
+
+Other covered content MAY also be included.
+
+If the singer's key presented is a JWK, the `keyid` parameter of the signature MUST be set
```

Comment:
singer -> signer ;-)

Reply:
No you see this method is only for musical APIs ;-)